src/routes/rows.js

```diff
@@ -3,37 +3,42 @@ const express = require('express');
 const controllers = require('../controllers/rows');
 const { broadcast } = require('../middlewares/broadcast');
 const { validator } = require('../middlewares/validation');
+const { processRequest, processResponse } = require('../middlewares/api');
 const schema = require('../schemas/rows');
 
 const router = express.Router();
 
 router.get(
   '/:name/rows',
   validator(schema.listTableRows),
-  controllers.listTableRows
+  processRequest,
+  controllers.listTableRows,
+  processResponse,
 );
 router.post(
   '/:name/rows',
   validator(schema.insertRowInTable),
+  processRequest,
   controllers.insertRowInTable,
-  broadcast
+  broadcast,
 );
 router.get(
   '/:name/rows/:pks',
   validator(schema.getRowInTableByPK),
-  controllers.getRowInTableByPK
+  controllers.getRowInTableByPK,
+  processResponse,
 );
 router.put(
   '/:name/rows/:pks',
   validator(schema.updateRowInTableByPK),
   controllers.updateRowInTableByPK,
-  broadcast
+  broadcast,
 );
 router.delete(
   '/:name/rows/:pks',
   validator(schema.deleteRowInTableByPK),
   controllers.deleteRowInTableByPK,
-  broadcast
+  broadcast,
```

Comment:
Shouldn't we use the process request and response here?

Reply:
Those middleware functions are added in the next PR